javascript/javascript3/week1/Weather_App/script.js

```diff
@@ -0,0 +1,96 @@
+(function () {
```

Comment:
Hi Mehmet! Great app you made. I just tested it locally and it works great!
I have no comments to do :) 

The only thing I could add to try and help with this code review is mentioning that it is important to start thinking about code splitting for the future. The idea is to try and split the logic into different files, to avoid doing "too much" in one file. This was not part of the exercise and you should not change your code, it is just a comment for future projects you will be making.

![image](https://user-images.githubusercontent.com/80256770/208647975-4838bec3-c610-415c-b4dc-3bea95cf2a0f.png)

You could think about whether it makes sense to organize your code in different files (for example map and weather logic), export the functions from each individual file, and then call them from your main.js which would work as a point of entry for your app.

Reply:
thank you so much for the tips, i'll try to use them for the next hw!